timeline_examples.html

```diff
@@ -1,129 +1,374 @@
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/other/performance.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/other/performance.png">
+          <div class="exampleTitle">performance</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/other/rtl.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/other/rtl.png">
+          <div class="exampleTitle">rtl</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/other/timezone.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/other/timezone.png">
+          <div class="exampleTitle">timezone</div>
+        </a>
+      </div>
+    </div>
+
+    <h3>styling</h3>
+    <div class="container-fluid">
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/styling/axisOrientation.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/styling/axisOrientation.png">
+          <div class="exampleTitle">exis orientation</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/styling/customCss.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/styling/customCss.png">
+          <div class="exampleTitle">custom css</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/styling/gridStyling.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/styling/gridStyling.png">
+          <div class="exampleTitle">grid styling</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/styling/itemClassNames.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/styling/itemClassNames.png">
+          <div class="exampleTitle">item class names</div>
+        </a>
+      </div>
+
+      <div class="col-lg-2 col-md-3 col-sm-6">
+        <a href="examples/timeline/styling/itemTemplates.html">
+          <img class="timeline_example img-responsive" src="./images/exampleScreenshots/timeline/styling/itemTemplates.png">
+          <div class="exampleTitle">item templates</div>
+        </a>
+      </div>
     </div>
-</div>
 
-<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-<!-- Include all compiled plugins (below), or include individual files as needed -->
-<script src="js/bootstrap.min.js"></script>
-</body>
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
```

Comment:
Are there any bootstrap plugin used? Mayba we can remove jquery and the boostrap.js file!?